config/deploy.rb

```diff
@@ -13,3 +13,4 @@
 
 set :linked_files, fetch(:linked_files, []).push('config/database.yml', 'config/secrets.yml')
 set :linked_dirs, fetch(:linked_dirs, []).push('log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'vendor/bundle', 'public/system', 'public/uploads', 'public/data')
+append :linked_dirs, '.bundle'
```

Comment:
I am not 100% sure what this line does, and we don't seem to be using  it in other projects. Is it needed?

Reply:
This is a capistrano optimization. Some background on when it was introduced and why here: https://github.com/capistrano/bundler/issues/96